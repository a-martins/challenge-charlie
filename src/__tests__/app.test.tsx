import { render } from "@testing-library/react";
import RouterProvider from "../routes/Router";

test("renders app component", () => {
  const { container, getByRole } = render(<RouterProvider />);

  expect(getByRole("main")).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
