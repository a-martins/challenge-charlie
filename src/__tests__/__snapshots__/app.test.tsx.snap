// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app component 1`] = `
<div>
  <div
    class="sc-aYaIB fVpkfD"
    id="app"
    role="main"
  >
    Weather App!
    <p>
      latitude: 
    </p>
    <p>
      longitude: 
    </p>
  </div>
</div>
`;
